input/fsh/profiles/RORHealthcareService.fsh

```diff
@@ -96,6 +93,16 @@ Description: "Profil créé dans le cadre du ROR pour décrire les prestations q
 * characteristic[temporalityStopService] ^short = "typeFermeture (OffreOperationnelle) : Temporalité d'arrêt de réalisation de la prestation"
 * characteristic[temporalityStopService] from $JDV-J185-TypeFermeture-ROR (required)
 
+* characteristic[act] ^slicing.discriminator[+].type = #value
```

Comment:
un warn sushi warn  HealthcareService.characteristic[act]: An element with a slice name should not define its own slicing. Instead, append additional discriminators to the original slicing on the base element.
  File: /home/runner/work/IG-fhir-repertoire-offre-ressources-sante/IG-fhir-repertoire-offre-ressources-sante/ig/input/fsh/profiles/RORHealthcareService.fsh

Reply:
@paulineheurtebise si c'est traité n'hésite pas à résoudre la conversation et à re-request ma review :)